proto/datastream/v1/datastream.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package datastream.v1;
+
+option go_package = "github.com/0xPolygonHermez/zkevm-node/state/datastream";
+
+message Batch {
+    uint64 number = 1;
+    bytes local_exit_root = 2;
+    bytes state_root = 3;
+    uint64 fork_id = 4;
+    uint64 chain_id = 5;
+}
+
+message L2Block {
```

Comment:
Now we have a hardcoded `BLOCK_GAS_LIMIT`. So, all blocks has exactly the same value.
Not sure If we are going to apply dynamic BLOCK_GAS_LIMIT in the future.
Just leaving open this question for disussion

Reply:
Maybe add `block_info_root`...which is computed in L2.
It maybe useful since it is kind of a summary of all transactions, its execution and block header parameters.

It is already embedded in `state_root` but it will be useful to see a match in this field for all clients implementing datastream. We already realize that this tree computation could be an issue so adding here would kind of a sanity check

Reply:
discussed AFK: add a `Debug` message that contains useful information to be ale to debug easily.
Only used with debugging purposes. Standard behavior should be set this `Debug` message to `empty`.

If the datastream is build with `debug = true`, it will fill all the information required in `Debug` message